Examples/Segmentation/HoughTransform2DLinesImageFilter.cxx

```diff
@@ -282,8 +282,8 @@ main(int argc, char * argv[])
 
     for (auto i = static_cast<int>(-diag); i < static_cast<int>(diag); ++i)
     {
-      localIndex[0] = static_cast<long int>(u[0] + i * v[0]);
-      localIndex[1] = static_cast<long int>(u[1] + i * v[1]);
+      localIndex[0] = static_cast<long>(u[0] + i * v[0]);
+      localIndex[1] = static_cast<long>(u[1] + i * v[1]);
```

Comment:
I did it in only `static_cast` situations precisely to avoid having this pull request wander into other territory.  It could get complicated in a wider scope and I didn't want to attempt that here.  However, I can undo any of the commits; please let me know.

Reply:
My preference would be to have a separate PR dealing with all those "optional ints", not just in the context of casting. Which could be either before or after this PR. It might trigger some discussion about style. But it would be clearly a separate discussion from the casting issue.

Reply:
That works for me.  I have removed the commit with this change.

Reply:
@Leengit Would you still want to make a separate PR to address overly verbose type specifiers like `signed long` or `long int` throughout the entire code base? If so, I would like it very likely  😃